src/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -10,10 +9,20 @@ import {DiskSpace} from "../DiskSpace/DiskSpaceContainer";
 import classNames from "classnames";
 import {DealsContainer} from "../Deals/DealsContainer";
 import {PledgedCollateralContainer} from "../PledgedCollateral/PledgedCollateralContainer";
+import {useSelector} from "react-redux";
+import {RootState} from "../../app/rootReducer";
+import {useHistory} from "react-router-dom";
+import {Routes} from "../../constants/routes";
 
 export const DashboardContainer = (): ReactElement => {
     const [areElementsHidden, setElementsHidden] = useState(false);
+    const history = useHistory();
+    const stateToken = useSelector((state: RootState) => state.login.token);
+    const storageToken = localStorage.getItem("token");
```

Comment:
This should definitely not be here. You can have an initial state loading from local storage.